scripts/resume.py

```diff
@@ -133,18 +133,26 @@ def instance_properties(partition, model, placement_group, labels=None):
             "values": [reservation_name],
         }
 
-        placement = util.reservation_placement(reservation)
-        if placement:
-            props.resourcePolicies = [placement]
+        policies = util.reservation_resource_policies(reservation)
+        if policies:
+            props.scheduling = {
+                "onHostMaintenance": "TERMINATE",
+                "automaticRestart": False,
+            }
+            props.resourcePolicies = policies
```

Comment:
I think this is probably the right choice (attach all policies). Are you aware of any  "bad" policies that might be bad to add bulkInsert call.

Even if such policies exist, I think we may describe that as a problem with the reservation than as a problem with the resume.py code. We just need to document the behavior as covering all resource policies, not just placement policies.

Reply:
I believe I removed any notion `**placement**_policy`, now it's general `**resource**_policy`.